all/remote/src/main/resources/features.xml

```diff
@@ -2,5 +2,6 @@
 <features name="infinispan-remote-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.0.0">
    <feature name="infinispan-remote" version="${project.version}">
       <bundle>mvn:org.infinispan/infinispan-remote/${project.version}</bundle>
+      <bundle dependency="true">mvn:org.apache.logging.log4j/log4j-api/${version.log4j}</bundle>
```

Comment:
Yes, this is the error you get if you omit it:

<details>
  <summary>Click to expand</summary>
```
[WARNING] Feature resolution failed for [infinispan-remote/9.0.0.SNAPSHOT]
Message: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=infinispan-remote; type=karaf.feature; version=9.0.0.SNAPSHOT; filter:="(&(osgi.identity=infinispan-remote)(type=karaf.feature)(version>=9.0.0.SNAPSHOT))" [caused by: Unable to resolve infinispan-remote/9.0.0.SNAPSHOT: missing requirement [infinispan-remote/9.0.0.SNAPSHOT] osgi.identity; osgi.identity=org.infinispan.remote; type=osgi.bundle; version="[9.0.0.SNAPSHOT,9.0.0.SNAPSHOT]"; resolution:=mandatory [caused by: Unable to resolve org.infinispan.remote/9.0.0.SNAPSHOT: missing requirement [org.infinispan.remote/9.0.0.SNAPSHOT] osgi.wiring.package; filter:="(osgi.wiring.package=org.apache.logging.log4j)"]]
Repositories: {
	file:///data/repositories/github/jboss/infinispan/all/remote/target/classes/features.xml
	mvn:org.apache.karaf.features/enterprise/3.0.8/xml/features
	mvn:org.apache.karaf.features/framework/3.0.8/xml/features
	mvn:org.ops4j.pax.cdi/pax-cdi-features/0.11.0/xml/features
}
Resources: {
	file:///data/repositories/github/jboss/infinispan/all/remote/target/classes/features.xml
	mvn:jline/jline/2.13
	mvn:org.apache.aries.blueprint/org.apache.aries.blueprint.api/1.0.1
	mvn:org.apache.aries.blueprint/org.apache.aries.blueprint.cm/1.0.8
	mvn:org.apache.aries.blueprint/org.apache.aries.blueprint.core.compatibility/1.0.0
	mvn:org.apache.aries.blueprint/org.apache.aries.blueprint.core/1.6.1
	mvn:org.apache.aries.proxy/org.apache.aries.proxy.api/1.0.1
	mvn:org.apache.aries.proxy/org.apache.aries.proxy.impl/1.0.4
	mvn:org.apache.aries.quiesce/org.apache.aries.quiesce.api/1.0.0
	mvn:org.apache.aries/org.apache.aries.util/1.1.1
	mvn:org.apache.felix/org.apache.felix.configadmin/1.8.4
	mvn:org.apache.felix/org.apache.felix.fileinstall/3.5.2
	mvn:org.apache.karaf.bundle/org.apache.karaf.bundle.command/3.0.8
	mvn:org.apache.karaf.bundle/org.apache.karaf.bundle.core/3.0.8
	mvn:org.apache.karaf.deployer/org.apache.karaf.deployer.blueprint/3.0.8
	mvn:org.apache.karaf.deployer/org.apache.karaf.deployer.features/3.0.8
	mvn:org.apache.karaf.deployer/org.apache.karaf.deployer.spring/3.0.8
	mvn:org.apache.karaf.deployer/org.apache.karaf.deployer.wrap/3.0.8
	mvn:org.apache.karaf.features/enterprise/3.0.8/xml/features
	mvn:org.apache.karaf.features/framework/3.0.8/xml/features
	mvn:org.apache.karaf.features/org.apache.karaf.features.command/3.0.8
	mvn:org.apache.karaf.features/org.apache.karaf.features.core/3.0.8
	mvn:org.apache.karaf.jaas/org.apache.karaf.jaas.config/3.0.8
	mvn:org.apache.karaf.jaas/org.apache.karaf.jaas.modules/3.0.8
	mvn:org.apache.karaf.region/org.apache.karaf.region.core/3.0.8
	mvn:org.apache.karaf.service/org.apache.karaf.service.guard/3.0.8
	mvn:org.apache.karaf.shell/org.apache.karaf.shell.commands/3.0.8
	mvn:org.apache.karaf.shell/org.apache.karaf.shell.console/3.0.8
	mvn:org.apache.karaf.shell/org.apache.karaf.shell.help/3.0.8
	mvn:org.apache.karaf.shell/org.apache.karaf.shell.table/3.0.8
	mvn:org.apache.karaf.system/org.apache.karaf.system.command/3.0.8
	mvn:org.apache.karaf.system/org.apache.karaf.system.core/3.0.8
	mvn:org.apache.sshd/sshd-core/0.14.0
	mvn:org.infinispan/infinispan-remote/9.0.0-SNAPSHOT
	mvn:org.jledit/core/0.2.1
	mvn:org.ops4j.pax.cdi/pax-cdi-features/0.11.0/xml/features
	mvn:org.ops4j.pax.logging/pax-logging-api/1.8.4
	mvn:org.ops4j.pax.logging/pax-logging-service/1.8.4
	mvn:org.ops4j.pax.url/pax-url-aether/2.4.7
	mvn:org.ops4j.pax.url/pax-url-wrap/2.4.7/jar/uber
	mvn:org.ow2.asm/asm-all/5.0.3
}
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```
</details>